client/src/components/Auth/SignUp.jsx

```diff
@@ -4,11 +4,13 @@ import { Link, withRouter } from 'react-router-dom';
 import { withFirebase } from '../../Firebase';
 import * as ROUTES from '../../Routes/routes';
 
+import { Form, FormField, Button, Box, Text, Heading } from 'grommet';
+
 const SignUpPage = () => (
-    <div>
-        <h1>SignUp</h1>
+    <Box align="center" background="#555555" height="100vh" pad="large">
```

Comment:
should these be hardcoded? I noticed the use of themes earlier. Is there a theme that could be passed into this box component?

Reply:
This is a part of the Grommet theme. Box is pulled from grommet and is a specific style.